GameServer/commands/gmcommands/adviceTeam.cs

```diff
@@ -29,15 +39,16 @@ public void OnCommand(GameClient client, string[] args)
 			foreach (GameClient playerClient in WorldMgr.GetAllClients())
 			{
 				if (playerClient.Player == null) continue;
-				if ((playerClient.Player.Realm == client.Player.Realm ||
-				     playerClient.Account.PrivLevel > 1) && !playerClient.Player.IsIgnoring(client.Player))
+				
+				if (playerClient.Player.Realm == client.Player.Realm && playerClient.Account.PrivLevel > 1)
```

Comment:
All players should be able to receive an /adviceteam message.

Reply:
Can't recall why I decided to keep that plvl check...